backend/corpora/lambdas/api/v1/authentication.py

```diff
@@ -47,7 +48,8 @@ def get_oauth_client(config: CorporaAuthConfig) -> FlaskRemoteApp:
         refresh_token_url=config.api_token_url,
         access_token_url=config.api_token_url,
         authorize_url=config.api_authorize_url,
-        client_kwargs={"scope": "openid profile email offline_access"},
+        client_kwargs={"scope": "openid profile email offline_access read:collections write:collections"},
```

Comment:
did we want to get rid of read:collections?

Reply:
I was thinking to leave it for future implementations, but we can always add it back later. Will remove it altogether.